src/s1reader/s1_reader.py

```diff
@@ -649,6 +650,12 @@ def burst_from_xml(annotation_path: str, orbit_path: str, tiff_path: str,
                                                                          aux_cal_subswath,
                                                                          sensing_start)
 
+        # Extenced FM and DC coefficient information
+        extended_coeffs = \
+            BurstExtendedCoeffs.from_product_annotation_and_burst(product_annotation,
```

Comment:
Do we need to load the extended coefficient every time we are reading a burst? I am concerned that we are making the reader a bit slower reading metadata which might be not necessary for other workflows (e.g. RTC?)

Reply:
Good point Maybe we can add a flag to turn this on / off, like we did for EAP correction and AUX_CAL loader?

Reply:
Added a flag `flag_az_fmrate_mismatch_mitigation` to turn on / off the feature.

@vbrancat @LiangJYu Any suggestions for the flag name above? It looks too long to me.